packages/botonic-plugin-google-analytics/README.md

```diff
@@ -0,0 +1,49 @@
+# Botonic Plugin Google Analytics
+
+## What Does This Plugin Do?
+
+With this plugin you can track the user interaction with the bot or the bot's behaviour sending the information to [Google Analytics](https://analytics.google.com/). 
+
+## Setup
+
+1. Run `npm install --save @botonic/plugin-google-analytics` to install the plugin.
+2. Add it to the `src/plugins.js` file:
+
+**src/plugins.js**
+```javascript
+export const plugins = [
+  {
+    id: 'google_analytics',
+    resolve: require('@botonic/plugin-google-analytics'),
+    options: {
+      botName: 'YOUR_BOT_NAME',
+      trackingId: 'YOUR_TRACKING_ID'
+    }
+  }
+]
+```
+
+## Use
+
+The tracking needs to be done manually, enabling this plugin doesn't track any user interaction or bot's behaviour by default.
+You can do the tracking inside the `botonicInit` method, `render` method or even inside the [Webchat listeners](https://docs.botonic.io/concepts/webchat#webchat-listeners)
```

Comment:
i'm not pretty sure about the usage inside the listeners itself (I've never used a plugin in this way), have you tried it @asastre ?

Reply:
@dpinol maybe is better to remove the `async` `await` from the `track` method because it doesn't make sense to stop the execution of the bot while doing the tracking and hence it could be called anywhere (`botonicInit`, `render` or webchat listeners)

Reply:
@vanbasten17 I've tryed to call it inside the `onInit` method and it works. I'll use it here to track the impressions of the chatbot for example.

Reply:
> @dpinol maybe is better to remove the `async` `await` from the `track` method because it doesn't make sense to stop the execution of the bot while doing the tracking and hence it could be called anywhere (`botonicInit`, `render` or webchat listeners)

You should not remove async, await. Or if you do, you should return the promise. Otherwise:
* you don't let the bot wait for the response in case they want to do it
* in massimo's bot where we track to dynamo, if you send the track when the lambda instance is restarted (it happens every few hours), then if you don't have the await, the track is never sent. I guess here would happen the same